SwiftLeeds/Views/About/SponsorGridView.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  SponsorGridView.swift
+//  SwiftLeeds
+//
+//  Created by Alex Logan on 01/07/2022.
+//
+
+import SwiftUI
+
+struct SponsorGridView: View {
+    @Environment(\.openURL) var openURL
+    let goldSponsors = Sponsor.platinum
```

Comment:
Why are gold sponsors using platinum instead of gold?

Reply:
Not even used, left-over from hacking away 👍🏻